src/creatures/creature.cpp

```diff
@@ -998,8 +998,10 @@ BlockType_t Creature::blockHit(std::shared_ptr<Creature> attacker, CombatType_t
 		attacker->onAttackedCreatureBlockHit(blockType);
 	}
 
-	mitigateDamage(combatType, blockType, damage);
-
+	if (damage != 0) {
```

Comment:
If the condition is the same to execute onTakeDamage, why not check the condition just once for both?

Reply:
Ye, in this moment it should be better, but when onTakeDamage in future will able to edit blokctype it shoudn't be in same condition.
I change it. Thanks for suggestion